app/qml/MainPanel.qml

```diff
@@ -28,13 +28,20 @@ Item {
     property color gpsIndicatorColor: InputStyle.softRed
 
     id: mainPanel
+    focus: true
 
     Rectangle {
         anchors.fill: parent
         color: InputStyle.clrPanelBackground
         opacity: InputStyle.panelOpacity
     }
 
+    Keys.onReleased: {
+      if (event.key === Qt.Key_Back || event.key === Qt.Key_Escape) {
+        // if main panel has focus - nothing else is opened, back button closes app
```

Comment:
not sure, this function does nothing, maybe you can remove it all? or it has missing function call?

Reply:
True, it does nothing now, however I kept it there for "future" use case. It is last place we handle the back button before app is closed. But we do not need it, I can remove it.